apps/storybook.namekit.io/stories/Namekit/CurrencySymbol.stories.tsx

```diff
@@ -0,0 +1,105 @@
+import { Currency } from "@namehash/ens-utils";
+import { CurrencySymbol, CurrencySymbolSize } from "@namehash/namekit-react";
+import type { Meta, StoryObj } from "@storybook/react";
+
+export const ETH: Story = {
+  args: {
+    currency: Currency.Eth,
+    size: CurrencySymbolSize.Large,
+  },
+};
+export const USD: Story = {
+  args: {
+    currency: Currency.Usd,
+    size: CurrencySymbolSize.Large,
+  },
+};
+export const USDC: Story = {
+  args: {
+    currency: Currency.Usdc,
+    size: CurrencySymbolSize.Large,
+  },
+};
+export const WETH: Story = {
+  args: {
+    currency: Currency.Weth,
+    size: CurrencySymbolSize.Large,
+  },
+};
+export const DAI: Story = {
+  args: {
+    currency: Currency.Dai,
+    size: CurrencySymbolSize.Large,
+  },
+};
+export const SmallSize: Story = {
+  args: {
+    size: CurrencySymbolSize.Small,
+    currency: Currency.Eth,
+  },
+};
+export const LargeSize: Story = {
+  args: {
+    size: CurrencySymbolSize.Large,
+    currency: Currency.Eth,
+  },
+};
+export const WithCustomSymbolColor: Story = {
+  args: {
+    symbolFillColor: "#007bff",
+    size: CurrencySymbolSize.Large,
+    currency: Currency.Eth,
+  },
+};
+export const ShowingTooltipDescription: Story = {
```

Comment:
This was related to [this](https://blockful.slack.com/archives/C03JQ5URTEG/p1723667208905799?thread_ts=1723482012.592049&cid=C03JQ5URTEG)... I was waiting for an answer about wether to do it in my branch or not but since you raised it here, I will fix it.